strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/ProxyRepositoryArtifactResolver.java

```diff
@@ -98,8 +98,8 @@ public InputStream getInputStream(final String storageId,
 
             is = onSuccessfulProxyRepositoryResponse(is, storageId, repositoryId, path);
 
-            final RepositoryPath artifactPath = layoutProvider.resolve(repository).resolve(path);
-            final RepositoryFileAttributes artifactFileAttributes = (RepositoryFileAttributes) Files.readAttributes(
+            RepositoryPath artifactPath = layoutProvider.resolve(repository).resolve(path);
+            RepositoryFileAttributes artifactFileAttributes = (RepositoryFileAttributes) Files.readAttributes(
```

Comment:
What's the purpose of this change ?

Reply:
hm... and why we need `final` here? :)

Reply:
> hm... and why we need final here? :)

we can debate one day on the purpose of keeping variables `final` or not, but the point here is that this change does not bring anything to this PR and definitely is not needed in terms of finishing SB-906. For me, it's just a noise reviewing PR.